src/widgets/heightandweight/heightandweight-util.tsx

```diff
@@ -0,0 +1,14 @@
+import { newWorkspaceItem } from "@openmrs/esm-api";
+
+export function openHeightAndWeightTab(addComponent, componentName): void {
```

Comment:
Hehe - the `void` is known anyway (we don't return anything), but `addComponent` and `componentName` still need a type.

Reply:
Surprised - I thought I got rid of this file in a previous commit. Must have rebased improperly. `openHeightAndWeightTab` should be replaced entirely by the reusable (and correctly typed) `openWorkspaceTab` function. I'll fix this.